pkg/pipeline/decode/decode_aws.go

```diff
@@ -51,11 +53,12 @@ func (c *decodeAws) Decode(in []interface{}) []config.GenericMap {
 	out := make([]config.GenericMap, 0)
 	nItems := len(in)
 	log.Debugf("nItems = %d", nItems)
-	for _, line := range in {
+	for i, line := range in {
```

Comment:
can you use significant name and not `i`

Reply:
OK